client/cloudevents/types/types.go

```diff
@@ -19,6 +19,14 @@ const (
 	SourceAll = ""
 )
 
+const (
+	// CloudEventsDataTypeAnnotationKey is the key of the cloudevents data type annotation.
+	CloudEventsDataTypeAnnotationKey = "cloudevents.open-cluster-management.io/datatype"
```

Comment:
what's this annotation for?

Reply:
this will be used by work-agent to distinguish a manifestwork's event data type, move to work package